modules/http2/h2_conn_ctx.h

```diff
@@ -48,20 +50,17 @@ struct h2_conn_ctx_t {
     struct h2_bucket_beam *beam_out; /* c2: data out, created from req_pool */
     struct h2_bucket_beam *beam_in;  /* c2: data in or NULL, borrowed from request stream */
 
-    apr_pool_t *mplx_pool;           /* c2: an mplx child pool for safe use inside mplx lock */
     apr_file_t *pipe_in_prod[2];     /* c2: input produced notification pipe */
-    apr_file_t *pipe_in_drain[2];    /* c2: input drained notification pipe */
     apr_file_t *pipe_out_prod[2];    /* c2: output produced notification pipe */
 
-    apr_pollfd_t pfd_in_drain;       /* c2: poll pipe_in_drain output */
     apr_pollfd_t pfd_out_prod;       /* c2: poll pipe_out_prod output */
 
     int has_final_response;          /* final HTTP response passed on out */
     apr_status_t last_err;           /* APR_SUCCES or last error encountered in filters */
-    struct h2_response_parser *parser; /* optional parser to catch H1 responses */
 
-    volatile int done;               /* c2: processing has finished */
+    apr_uint32_t started;            /* c2: processing was started */
     apr_time_t started_at;           /* c2: when processing started */
+    apr_uint32_t done;               /* c2: processing has finished */
```

Comment:
Maybe leave the `volatile` indicator for `started` and `done` to document that they are accessed atomically (now explicitely) still?

Reply:
good idea, will add the `/* volatile */` comments.

Reply:
+1

Reply:
added